Resources/Prototypes/Reagents/gases.yml

```diff
@@ -0,0 +1,125 @@
+﻿- type: reagent
+  id: Oxygen
+  name: oxygen
+  desc: An oxidizing, colorless gas.
+  physicalDesc: gaseous
+  color: "#808080"
+  boilingPoint: -183.0
+  meltingPoint: -218.4
+  metabolisms:
+    Gas:
+      effects:
+      - !type:Oxygenate
+        conditions:
+        - !type:OrganType
+          type: Human
+      - !type:Oxygenate
+        conditions:
+        - !type:OrganType
+          type: Animal
+      - !type:CreateLungGas
+        ratios:
+          CarbonDioxide: 1.0
+
+- type: reagent
+  id: Plasma
+  name: plasma
+  desc: Funky, space-magic pixie dust. You probably shouldn't eat this, but we both know you will anyways.
+  physicalDesc: gaseous
+  color: "#7e009e"
+  boilingPoint: -127.3 # Random values picked between the actual values for CO2 and O2
+  meltingPoint: -186.4
+  tileReactions:
+  - !type:FlammableTileReaction
+    temperatureMultiplier: 1.5
+  metabolisms:
+    Poison:
+      effects:
+      - !type:HealthChange
+        damage:
+          types:
+            Poison: 3
+      - !type:AdjustReagent
+        reagent: Inaprovaline
+        amount: -2.0
+    Gas:
+      effects:
+      - !type:HealthChange
+        scaleByQuantity: true
+        ignoreResistances: true
+        damage:
+          types:
+            Poison:
+              1
+      - !type:CreateLungGas
+        ratios:
+          Plasma: 1.0
```

Comment:
I think create lung gas is fine for things like oxygen -> CO2, but think basing the general process of lung-reagents reverting to gasses on metabolisms is weird. I think it makes more sense to just revert the gas-to-reagent function and turn all present reagents back into gasses.

Currently, if you are in a high-pressure environment (or just set internals to high release pressure), you take in more moles/units of reagent than you can metabolize back into gas and your lungs flood with reagent. It just seems weird to me. After exhaling, the lungs should be empty, gasses shouldn't stick around.

I guess a workaround would be to just lower the lung's solution volume, to ensure that it can **always** metabolise every reagent inside of it in a single tick?

Reply:
i'll think about it a little bit. maybe something like

turn all gas into reagent -> process all reagents at once, but remove none of them (if a gas like oxygen needs to be removed and turn into co2, it should do that itself) -> turn reagents into gases and exhale?

Reply:
So basically adding a `RemoveLungGas` reagent effect and then also emptying the lung solution at the end? I guess that would work.